CONTRIBUTING.md

```diff
@@ -63,15 +63,19 @@ Alternately you can configure a compatible test server as follows:
 
 You should now be able to execute the tests, from project root folder:
 
-    `npm test`
+    ./vendor/bin/phpunit
+
+You may also run tests directly using phpunit as follows:
+
+    npm test
 
 Make sure your code is linted with phpcs (PSR-2 Coding Style)
```

Comment:
I figure just so people aren't wondering about why we would enforce PSR-2 we should make this a link. You can use http://www.php-fig.org/psr/psr-2/, which is the style guide published by the framework interop group. 

Making it...
Make sure your code is linted with phpcs ([PSR-2 Coding Style](http://www.php-fig.org/psr/psr-2/))